src/app/content/highlights/components/SummaryPopup/ChapterFilter.tsx

```diff
@@ -108,7 +108,7 @@ export default styled(ChapterFilter)`
   ${theme.breakpoints.mobile(css`
     &&& {
       left: -${mobilePadding.leftRight}rem;
-      max-width: calc(100vw - ${mobileMargin}rem * 2);
+      max-width: calc(100vw - ${mobileMarginSides}rem * 2);
```

Comment:
This will most likely not change anything, but I think it would nicer as:  `max-width: calc(100vw - ${mobileMarginSides * 2}rem );`

Reply:
i also prefer doing math in js rather than css when possible

Reply:
sorry, I fixed this in the re fetching highlights PR